sentry_sdk/integrations/loguru.py

```diff
@@ -11,12 +12,15 @@
 
 if TYPE_CHECKING:
     from logging import LogRecord
-    from typing import Optional, Any
+    from typing import Any, Optional, Tuple
```

Comment:
[nit] Should be possible to to subscript the built-in `tuple` (see corresponding suggestion below)

```suggestion
    from typing import Any, Optional
```